src/fides/api/ops/api/v1/endpoints/privacy_experience_endpoints.py

```diff
@@ -81,56 +70,34 @@ def privacy_experience_list(
             PrivacyExperience.disabled.is_(False)
         )
     if region is not None:
-        experience_query = experience_query.filter(
-            PrivacyExperience.regions.contains([region])
-        )
+        experience_query = experience_query.filter(PrivacyExperience.region == region)
     if component is not None:
         experience_query = experience_query.filter(
             PrivacyExperience.component == component
         )
+    if has_config is True:
+        experience_query = experience_query.filter(
+            PrivacyExperience.experience_config_id.isnot(None)
+        )
+    if has_config is False:
+        experience_query = experience_query.filter(
+            PrivacyExperience.experience_config_id.is_(None)
+        )
```

Comment:
I think it would be a really great addition if we had "default" ExperienceConfigs - one for a banner overlay, link overlay, and privacy center that we could link to automatically on the user's behalf, but right now we don't have these in the system. So it's possible we created an Experience that doesn't have any language associated (the config).  

These filters let us surface with PrivacyExperiences have a config (and can be surfaced to an end user) versus don't have a config (and need editing by the compliance person)